docs/tutorials/analysis/1D/spectral_analysis_hli.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>In "Proposed approach", "<code>Analysis</code> class to orchestrate data reduction" - it is not just data reduction, the full analysis is done with the Analysis class here, no?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/4044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>agree</p>